src/BenchmarksDriver/Program.cs

```diff
@@ -99,6 +99,8 @@ public static int Main(string[] args)
                 "Number of threads used by client", CommandOptionType.SingleValue);
             var headerOption = app.Option("--header",
                 "Header added to request", CommandOptionType.MultipleValue);
+            var methodOption = app.Option("-p|--method",
```

Comment:
I would remove the short option `-p`.  There are now too many options to choose reasonable single-letter shortcuts, so we might as well stop.